recipes/cuda-nvcc-impl/meta.yaml

```
@@ -0,0 +1,164 @@
+{% set name = "cuda-nvcc-impl" %}
+{% set version = "12.0.76" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvcc/{{ platform }}/cuda_nvcc-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 160ca9f8828f53daa4b2066c5361aacb45fa2575885f70c223cda1d11df53d6f  # [linux64]
+  sha256: dc5bd7eac57b32a9645c0c23c6a1ba8a9533e846eeec31414e1aa4230afee2f8  # [ppc64le]
+  sha256: 85c8c643ea2cfa398e1b5ffcd597f5bb3b738526c9e3fb8e39dd909c55345f36  # [aarch64]
+  sha256: f67406f44263f0e4a110a8cf0200bff1d3d0e32917db8a7c9e417a87eb7ca372  # [win]
+  patches:
+    - nvcc.profile.patch      # [linux]
+    - nvcc.profile.patch.win  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx or win]
+
+test:
+  commands:
+    - test -L $PREFIX/include/crt                    # [linux]
+    - test -L $PREFIX/include/fatbinary_section.h    # [linux]
+    - test -L $PREFIX/include/nvPTXCompiler.h        # [linux]
+    - test -L $PREFIX/lib/libnvptxcompiler_static.a  # [linux]
+
+outputs:
+  - name: cuda-nvcc-tools
+    files:
+      - targets/{{ target_name }}/bin/nvcc  # [linux]
+      - targets/{{ target_name }}/nvvm      # [linux]
+      - bin                                 # [linux]
+      - nvvm                                # [linux]
```

Should only be the executable. We don't want the library here
```suggestion
      - nvvm/bin                                # [linux]
```